scripts/start-finalExec

```diff
@@ -285,6 +285,9 @@ EOF
   if isTrue "${EXEC_DIRECTLY:-false}"; then
     "${finalArgs[@]}"
   else
+    JVM_ARGS="${JVM_XX_OPTS} ${JVM_OPTS} $expandedDOpts"
+    JVM_ARGS=${JVM_ARGS//$'\n'/}
+    sed -i "s~JAVA_ARGS=.*~JAVA_ARGS=\"${JVM_ARGS}\"~" "${FTB_DIR}/variables.txt"
```

Comment:
Please add a conditional on variables.txt file existing. One big reason I deprecated support for server modpacks is that there was no guaranteed consistency. Some embed the equivalent in their start script.